pyrightconfig.json

```diff
@@ -2,7 +2,8 @@
     "typeshedPath": ".",
     "include": [
         "stdlib",
-        "stubs"
+        "stubs",
+        "test_cases"
```

Comment:
- Do we really want this to be in both pyrightconfigs?
- Does mypy support `assert_type()` yet? If it does, we could instead point `mypy_primer` at typeshed's `test_cases`. That way we would know if a PR to mypy breaks the tests.

Reply:
Mypy supports assert_type() on master (I added it last week). I am hoping to convince Jukka to put it in the upcoming 0.950 release. We should make mypy run on this directory too as soon as 0.950 is released, or use master if we're feeling brave.

Reply:
Update: `assert_type` has been cherry-picked onto the 0.950 branch, so we should be able to point mypy_primer at this directory as soon as 0.950 is released.

Reply:
Hopefully we can just run mypy regularly, no need to use mypy-primer.